app/views/landing-pages/league/PageLeagueGlobal.vue

```diff
@@ -270,7 +289,7 @@ export default {
         style="max-width: 800px; margin-bottom: 50px;"
       >The CodeCombat AI League is uniquely both a competitive AI battle simulator and game engine for learning real Python and JavaScript code.</p>
     </div>
-    <div class="row flex-row text-center">
+    <div v-if="!doneRegistering && !isClanCreator" class="row flex-row text-center">
```

Comment:
All of these `!isClanCreator` checks make it much more clear that you have successfully created a clan and are not on the same page any more.